doc/classes/ParticlesMaterial.xml

```diff
@@ -266,6 +281,36 @@
 		<member name="tangential_accel_min" type="float" setter="set_param_min" getter="get_param_min" default="0.0">
 			Minimum tangential acceleration.
 		</member>
+		<member name="turbulence_active" type="bool" setter="set_turbulence_active" getter="get_turbulence_active" default="false">
+			Enables and disables Turbulence for the particle system.
+		</member>
+		<member name="turbulence_influence_max" type="float" setter="set_param_max" getter="get_param_max">
+			Minimum turbulence influence on each particle.
+		</member>
+		<member name="turbulence_influence_min" type="float" setter="set_param_min" getter="get_param_min">
+			Maximum turbulence influence on each particle.
+		</member>
+		<member name="turbulence_influence_over_life" type="Texture2D" setter="set_param_texture" getter="get_param_texture">
+			Each particle's amount of turbulence will be influenced along this [CurveTexture] over its life time.
+		</member>
+		<member name="turbulence_initial_displacement_max" type="float" setter="set_param_max" getter="get_param_max">
+			Maximum displacement of each particles spawn position by the turbulence.
+		</member>
+		<member name="turbulence_initial_displacement_min" type="float" setter="set_param_min" getter="get_param_min">
+			Minimum displacement of each particles spawn position by the turbulence.
+		</member>
+		<member name="turbulence_noise_scale" type="float" setter="set_param_direct" getter="get_param_direct">
+			This value controls the overall scale of the turbulence noise pattern.
```

Comment:
```suggestion
			This value controls the overall scale of the turbulence noise pattern. In other noise systems, this value is referred to as frequency. A higher scale will result in smaller features with more detail while a lower scale will result in smoother noise with larger features.
```

Reply:
This description is not correct. scale is not frequency. A higher scale will create a "smoother" noise, a lower scale will create smaller features.

So basically the opposite of frequency.
I'd still put in the reference to frequency, but a bit less prominent as it is not quite the same:
`This value controls the overall scale/frequency of the turbulence noise pattern.
A small scale will result in smaller features with more detail while a high scale will result in smoother noise with larger features.`